index.html

```diff
@@ -61,7 +61,7 @@
   <meta name="twitter:creator" content="@{$ organizer.twitter $}">
   <meta name="twitter:title" content="{$ title $}">
   <meta name="twitter:description" content="{$ description $}">
-  <meta name="twitter:image" content="/images/social-share.jpg">
+  <meta name="twitter:image" content="{$ url $}images/social-share.jpg">
```

Comment:
Could you also add `{$ url $}` to other meta tags? (ex. https://github.com/gdg-x/hoverboard/blob/master/index.html#L45)

Reply:
Sure ;)